common/src/main/java/org/cloudfoundry/identity/uaa/authorization/UaaAuthorizationEndpoint.java

```diff
@@ -445,7 +453,7 @@ private String getUnsuccessfulRedirect(AuthorizationRequest authorizationRequest
             template.query(values.toString());
         }
 
-        return template.build().expand(query).encode().toUriString();
+        return template.build(true).toUriString();
```

Comment:
and we no longer expand variables, everything that the UriComponentsBuilder is already encoded